runtime/oti/shcdatatypes.h

```diff
@@ -30,14 +30,17 @@
 extern "C" {
 #endif
 
+#if defined(J9VM_ENV_DATA64)
+#define MULTI_LAYER_CACHE
+#endif /* defined(J9VM_ENV_DATA64) */
```

Comment:
Is this define going to be permanent or should this be a build flag? The corresponding JIT changes will have to be guarded the same way.

Reply:
I don't think we will enable this feature on 32-bit VM.
I am fine with a build flag or moving this to j9.h (so that JIT can access). Which one do you prefer ?

Reply:
I suggest putting it in `j9cfg.h.ftl` (and `j9cfg.h.in` for `cmake`).
Can make the name clearer? It should refer to 'shared class cache'; how about `J9VM_OPT_ MULTI_LAYER_SHARED_CLASS_CACHE `?

Reply:
New Macro `J9VM_OPT_MULTI_LAYER_SHARED_CLASS_CACHE` added to `j9cfg.h.ftl` and `j9cfg.h.in`